odoo/addons/base/tests/test_res_currency.py

```diff
@@ -112,9 +112,9 @@ def test_res_currency_name_search(self):
             {"name": "1971-01-01", "rate": 1.5, "currency_id": currency_B.id},
             {"name": "1972-01-01", "rate": 0.69, "currency_id": currency_B.id},
         ])
-        # should not try to match field 'rate' (float field)
-        self.assertEqual(self.env["res.currency"].search_count([["rate_ids", "=", "1971-01-01"]]), 2)
         # should not try to match field 'name' (date field)
+        self.assertEqual(self.env["res.currency"].search_count([["rate_ids", "=", "1971-01-01"]]), 2)
```

Comment:
Wait, you haven't changed anything in the test file, except moving lines.  How could this test your fix?

Reply:
Please add a proper test that covers your case.